app.js

```diff
@@ -11,62 +12,100 @@ const teamTwoSubtractButton = document.getElementById('team-two-subtract-button'
 const finishGameButton = document.getElementById('finish-game-button');
 const teamOneLabel = document.getElementById('team-one-name');
 const teamTwoLabel = document.getElementById('team-two-name');
+const teamOneInput = document.getElementById('team-one');
+const teamTwoInput = document.getElementById('team-two');
+
+
 
 // create an array to hold on to the state of past games
 
 let name1 = '';
-let name2 =  '';
+let name2 = '';
 let score1 = 0;
 let score2 = 0;
+let pastGames = [];
 
 nameFormButton.addEventListener('click', () => {
     // get the name data from the form
+    const teamOneName = teamOneInput.value;
+    teamOneLabel.textContent = teamOneName;
+    const teamTwoName = teamTwoInput.value;
+    teamTwoLabel.textContent = teamTwoName;
+    
 
     // set the state to this data from the form
-
+    name1 = teamOneInput.value;
+    name2 = teamTwoInput.value;
+    
+    
     // reset the form values
+    
+    teamOneInput.value = '';
+    teamTwoInput.value = '';
+    
+    
+    refreshCurrentGameEl();
 
-    // refresh the current game element with new data by calling the appropriate function
 });
 
 
 teamOneAddButton.addEventListener('click', () => {
-    // increment the current state for team one's score
     
-    // refresh the current game element with new data by calling the appropriate function
+    currentGameEl.textContent = '';
```

Comment:
It seems like this line is repeated here and on lines 64, 72, & 80. Is there a function that you can move this into? Perhaps a function whose only job is to refresh the currentGameEl?